### ISSUE TITLE
Improve test_mapping_script

### ISSUE THREAD

**Issue by eriktamsen** [BAMresearch/LebeDigital#84]
In PR https://github.com/BAMresearch/LebeDigital/pull/53 @joergfunger mentioned a couple of points to improve the test of the [mapping script](https://github.com/BAMresearch/LebeDigital/blob/main/tests/data_extraction/test_mapping_script.py). I will try to summarize the points and include Jörgs original comments:

**Line 10:** 

- rename mapping files
- use pytest not unittest package

> 
mapping is done for different experiments, in that case Youngs modulus test (which is called emodul_mapping). Thus, I would suggest to have a file in test/mapping that is called test_emodul_mapping.py). I would also suggest we stick to a single testing framework (as Erik has already pointed out) and use pytest instead of unittest.

**Line 13:**

- use loguru logger not print in tests

>
It would be advantageous to not use print but rather the loguru logger such that we can define (when running the tests) the level of output we are interested in.

**Line 53:**
- open question

> 
are these newly created files added to the gitignore

**Line 64:**

- change the query for values

>
This only checks of there is a single informationbearingentity1 with the value. How do we know that this is the diameter? I would think it might be relevant to start from the id of the test and then extract its diameter traversing the complete graph. This might look redundant (and does not allow to have a query_and_test method) but IMO advantageous (since this is how we are supposed to use the knowledge graph afterwards). This is true for all the subsequent tests.

**Comment by soudehMasoudian** [comment]
@alFrie yes, but I didnt commit my changes as Line 64 is not resolved yet. I need to change the query and then commit all the changes as all of them mentioned in one issue.

**Comment by eriktamsen** [comment]
I dont fully follow what is being developed where.
1) Is the work done in branch 84 which is linked to this issue still used? If not, close this issue, delete the branch. Otherwise, merge the branch, and this issue will be closed automatically.
2) what are branches "soudehMasoudian-patch-1" and "soudehMasoudian-patch-2"? Looks like it is something created automatically?
I would prefer to open a new issue if new work is done on the mapping, then create a new branch from there which is linked and has the same number and then create a pull request. That way everything stays organized.
